docs/components/modeler/web-modeler/connectors/available-connectors/slack.md

```diff
@@ -0,0 +1,92 @@
+---
+id: slack
+title: Slack connector
+description: Working with Slack connector
+---
+
+The **Slack connector** allows you to interact with a [Slack](https://slack.com/) workspace, for example by sending Slack messages from your BPMN processes.
+
+## Prerequisites
+
+To use the **Slack connector**, a Slack app must be registered with the Slack workspace you would like to send messages to, and a respective OAuth token needs to be configured as secret in your cluster. Follow [these steps in the appendix](#appendix) to learn how to set this up.
```

Comment:
Minor comment, possible to update the text to the following?
"To use the Slack connector, a Slack app must be registered with the Slack workspace you would like to send messages to. A respective OAuth token needs to be configured as a secret in your cluster. Follow these steps in the appendix to learn how to set this up."

Reply:
Makes sense. Updated.